Makefile

```diff
@@ -49,20 +49,20 @@ format:
 test: check test_python test_spark prove_spark
 
 test_python:
-	python3 -m pytest -vv
+	python3 -m pytest -n$(shell nproc) -vv
 
 test_python_optimized:
-	python3 -O -m pytest -vv
+	python3 -O -m pytest -n$(shell nproc) -vv
 
 test_python_coverage:
 	coverage run --branch --source=rflx -m pytest -vv
 
 test_spark: $(test-files)
-	gprbuild -P$(project)
+	gprbuild -P$(project) -j0
 	$(test-bin)
 
 test_spark_optimized: $(test-files)
-	gprbuild -P$(project) -Xoptimization=yes
+	gprbuild -P$(project) -Xoptimization=yes -j0
```

Comment:
Is there a reason for not adding this to the project file as we did for gnatprove?

Reply:
I didn't find any way to specify this inside the gpr file. The switches defined in the file are always passed to the called tool, so it would try to call `gcc` and `gnatbind` with `-j0` which doesn't work.

Reply:
Adding `for Switches ("Ada") use ("-j0")` to `package Builder` works for me.

Reply:
Interesting. The only documentation I found was using this in aggregate projects and stated

> These switches are only read from the main aggregate project (the one passed on the command line), and ignored in all other aggregate projects or projects.

Reply:
>GPRbuild takes into account switches that may be specified on the command line or in attributes Switches(\<main or language>) or Default_Switches (\<language>) in package Builder of the main project.


http://docs.adacore.com/live/wave/gprbuild/html/gprbuild_ug/gprbuild_ug/building_with_gprbuild.html#switches

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/296/commits/a9cd89b90df837a5118ac6e679ecc3d0786f855f.